app/root.module.css

```
@@ -0,0 +1,9 @@
+.root {
+  width: 50svw;
+  min-width: 20rem;
+  height: 100vh;
+  margin: 0 auto;
```

```suggestion
  margin: 0 auto;
  box-sizing: border-box;
```
このように画面全体に合わせたい場合, `box-sizing: broder-box;` とすれば期待したレイアウトになると思います.